app/auth/auth.py

```diff
@@ -115,3 +124,18 @@ async def get_authentication_context(
         yield context
     finally:
         auth_context.reset(reset_token)
+
+
+async def check_operations_account(
+    context: Annotated[AuthenticationContext, Depends(get_authentication_context)],
+):  # noqa: E501
+    """
+    This function ensures that the account type is of type OPERATIONS
+
+    """
+    if context.account.type != AccountType.OPERATIONS:
+        # This API can only be consumed in the context of an Operations Account
+        raise HTTPException(
+            status_code=status.HTTP_403_FORBIDDEN,
+            detail="You are not Allowed",
```

Comment:
This message can be improved. 
Suggestions:

Pro option: Insufficient permissions to complete this action.
Sexy option: You’ve found the door, but you don’t have the key.